backend/Makefile

```diff
@@ -1,5 +1,16 @@
 APP_NAME ?= backend
 IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME):$(DOCKER_TAG)
+# VERIFY_IGNORE is a grep pattern to exclude files and directories from verification
+VERIFY_IGNORE := /vendor\|/automock
+
+# DIRS_TO_CHECK is a command used to determine which directories should be verified
+DIRS_TO_CHECK = go list ./... | grep -v "$(VERIFY_IGNORE)"
+
+.PHONY: fmt-local
+fmt-local:
+	go fmt $$($(DIRS_TO_CHECK))
+
+release: fmt-local
```

Comment:
You're overriding release, so you also need to add all other targets here, otherwise only `fmt-local` would get executed:
`release: fmt-local docker-build docker-push cosign`

Also this line needs to be added below `include`, because order of definitions is important, currently this definition is overshadowed by the `common_makefile.mk` one